tock/tock/middleware.py

```diff
@@ -34,3 +34,15 @@ def __call__(self, request):
             datetime.now().strftime(fmt)
 
         return self.get_response(request)
+
+
+class NoCacheMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        response['Cache-Control'] = 'no-store'
+        response['Pragma'] = 'no-cache'
+        response['Expires'] = '0'
```

Comment:
Can we use `from django.views.decorators.cache import never_cache` instead?

https://docs.djangoproject.com/en/5.1/topics/http/decorators/#django.views.decorators.cache.never_cache

Reply:
or even better, `from django.utils.cache import add_never_cache_headers`